include/rnp/rnp.h

```diff
@@ -642,8 +656,28 @@ RNP_API rnp_result_t rnp_unload_keys(rnp_ffi_t ffi, uint32_t flags);
  *              RNP_LOAD_SAVE_BASE64 should set to allow import of base64-encoded keys (i.e.
  *              autocrypt ones). By default only binary and OpenPGP-armored keys are allowed.
  * @param results if not NULL then after the successful execution will contain JSON with
- *                information about new and updated keys. You must free it using the
- *                rnp_buffer_destroy() function.
+ *              information about new and updated keys. You must free it using the
+ *              rnp_buffer_destroy() function.
+ *              JSON output is an object containing array of objects named "keys".
+ *              Each array item is an object representing an imported key.
+ *              It contains the following members:\n
```

Comment:
This `\n` looks a bit strange through text, I guess it's the only way to make 'hard' line break?

Reply:
Yes, or HTML tag `<br>` which looks even worse I think.